typescript/src/deposit.ts

```diff
@@ -66,14 +72,14 @@ export interface Deposit {
  * @param bitcoinClient - Bitcoin client used to interact with the network.
  * @param witness - If true, a witness (P2WSH) transaction will be created.
  *        Otherwise, a legacy P2SH transaction will be made.
- * @returns Empty promise.
+ * @returns Deposit UTXO.
```

Comment:
Maybe we could add more information on what tis returned deposit really is? Something like:
`@returns The UTXO that will be created by the deposit transaction.`

Reply:
https://github.com/keep-network/tbtc-v2/pull/298/commits/416f2f55013e3a2d3684ee69ef9594a8022e7665